freqtrade/constants.py

```diff
@@ -21,6 +21,7 @@
 AVAILABLE_PAIRLISTS = ['StaticPairList', 'VolumePairList',
                        'PrecisionFilter', 'PriceFilter', 'SpreadFilter']
 AVAILABLE_DATAHANDLERS = ['json', 'jsongz']
+AVAILABLE_CANCELLATIONS = ['unfilled_orders', 'none']
```

Comment:
Any particular reason to use a enum here instead of a true/false flag?

i think that would simplify configuration from a user perspective... 

```json
{
  "cancel_open_orders_on_exit": true
}
```

I guess it's to keep further extensibility - but what else could we cancel eventually in the future?  (this is a serious question - please consider and answer before changing this, as the truthyness of my whole comment here depends on the answer ...)

Reply:
Yeah, so I did it like this because @hroff-1902 had suggested that we may want to have three options: `cancel_orders | closing_positions | keep_orders_and_positions` - I'm not sure if there's a real need for closing positions though? If I am understanding correctly the functionality could be achieved with a `/stopbuy` and `/forcesell` ? I can see the argument for a boolean here!

Reply:
Yeah exactly.

Also - closing positions is a VERY dangeous approach in my opinion - and will most likely prevent people from updating a bot to the latest version as it'll do a "hard stop" on all open orders.

Reply:
With that in mind, I have updated it to be a boolean, `cancel_open_orders_on_exit`